src/app/pages/links/LinkPage-LinkList.component.tsx

```diff
@@ -1,27 +1,85 @@
-import { List, ListSubheader } from "@mui/material";
-import React from "react";
+import React, { useEffect, useState } from "react";
 import { useTranslation } from "react-i18next";
-import links from "../../../assets/external_links.json";
-import { LinkListContentComponent } from "./LinkList-ListContent.component";
+import { List, ListSubheader } from "@mui/material";
+import axios from "axios";
+import { ListContentListItemComponent } from "./ListContent-ListItem.component";
+
+interface ExternalLink {
+    Name: string;
+    Link: string;
+    category:
+        | "legal regulation"
+        | "Reports"
+        | "Organizations and Institutes"
+        | "Online Tools";
+}
+
+interface ApiData {
```

Comment:
We already have interfaces for the API respons - CMSEntity and CMSResponse. Please use these